internal/operators/ocs/validations.go

```diff
@@ -110,6 +110,21 @@ func (o *ocsValidator) ValidateOCSRequirements(cluster *models.Cluster) bool {
 		return false
 	} else {
 		for _, host := range hosts { // if the worker nodes >=3 , install OCS on all the worker nodes if they satisfy OCS requirements
+
+			/* if number of hosts are >=6 and one of them has Role as Auto-assign,
+			it is a problem because we won't know that node would be a master or a worker as the role would be Auto assign
+			Hence without knowing that won't be able to run OCS validations
+			*/
+			if host.Role == models.HostRoleAutoAssign {
+				status = "Not supporting OCS installation if any 1 host has role Auto-assign for number of host >=6"
+				o.log.Info(status, "\nSetting Operator Status")
```

Comment:
please change it to o.log.Info("Setting Operator Status ", status) and please do it for lines https://github.com/openshift/assisted-service/blob/master/internal/operators/ocs/validations.go#L102